StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetLoader.swift

```diff
@@ -189,6 +190,51 @@ final class PaymentSheetLoader {
         return intent
     }
 
+    static func additionalParams(configuration: PaymentSheet.Configuration) -> [String: Any]? {
+        if let ephemeralKey = configuration.customer?.ephemeralKeySecret {
+            return ["legacy_customer_ephemeral_key": ephemeralKey]
+        }
+        return nil
+    }
+
+    static func getPaymentMethodsStrategy(intent: Intent, configuration: PaymentSheet.Configuration) -> Task<[STPPaymentMethod], Error> {
+        if let saved = getPaymentMethodsFrom(intent: intent) {
+            return Task {
+                saved
+            }
+        } else {
```

Comment:
Let's add a comment explaining that we fall back to fetching the saved PMs manually